index.bs

```diff
@@ -850,14 +850,14 @@ To <dfn>populate the pose</dfn> of an {{XRSpace}} |space| in an {{XRSpace}} |bas
   1. Let |transform| be |pose|'s {{XRPose/transform}}.
   1. Query the [=/XR device=]'s tracking system for |space|'s pose relative to |baseSpace| at the time represented by |frame|, then perform the following steps:
     <dl class="switch">
-      <dt> If the tracking system provides a [=6DoF=] pose:
+      <dt> If the tracking system provides an actively tracked or statically known [=6DoF=] pose for |space|'s pose relative to |baseSpace|:
```

Comment:
nit: maybe flip this around to make sure that the "actively tracked" part is clear to be 6DOF
>      <dt> If the tracking system provides a statically known or actively tracked [=6DoF=] pose for |space|'s pose relative to |baseSpace|:

Reply:
I went with the following, to match the flow of the next bullet:

> If the tracking system provides a [=6DoF=] pose whose position is actively tracked or statically known for |space|'s pose relative to |baseSpace|: